lib-php/website/reverse-only-search.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+require_once(CONST_LibDir.'/ParameterParser.php');
```

Comment:
This is missing at least: require_once(CONST_LibDir.'/init-website.php');
I get PHP errors when testing.

Reply:
set_exception_handler_by_format() also requires it, so had to add it. Not sure why I was not getting PHP errors.